source/application/BioXAS/BioXASAppController.cpp

```diff
@@ -154,6 +154,23 @@ void BioXASAppController::onRegionOfInterestBoundingRangeChanged(AMRegionOfInter
 		xasConfiguration_->setRegionOfInterestBoundingRange(region);
 }
 
+void BioXASAppController::goToBeamStatusView(AMControl *control)
```

Comment:
I feel this function name is not straight-forward. Maybe we should think about rename it?